themes/bootstrap3/templates/RecordDriver/SolrOverdrive/hold.phtml

```diff
@@ -116,144 +134,140 @@
         </div>
         <?php else: ?>
         <div class="alert alert-danger">
-            <?=$this->icon('overdrive-warning') ?>
             <div class="moreInfo"><?=$result->msg?></div>
         </div>
 
         <?php endif;?>
 
-    <?php elseif ($action == 'cancelHoldConfirm'): ?>
-            <form class="" method="POST" name="cancelHold" data-lightbox-onclose="VuFind.refreshPage">
-
-                <div class="form-group pull-right">
-                    <input name="doAction" value="cancelHold" type="hidden">
-                    <input class="btn btn-primary" name="cancelHold" value="<?=$this->transEscAttr('od_but_cancel_hold'); ?>" type="submit">
-                </div>
-            </form>
+    <?php elseif ($action == 'editHoldEmailConfirm'): ?>
+        <form class="" method="POST" name="editHoldEmail" data-lightbox-onclose="VuFind.refreshPage">
+            <div class="form-group">
+                <label class="control-label"><?=$this->transEsc('Email Address'); ?></label>
+                <input id="email" name="email" value="<?=$result->data->hold->emailAddress?>" size="" class="form-control" type="text">
 
+            </div>
+            <div class="form-group">
+                <input name="doAction" value="editHoldEmail" type="hidden">
+                <input class="btn btn-primary" name="editHoldEmail" value="<?=$this->transEscAttr('Submit'); ?>" type="submit">
+            </div>
+        </form>
 
-    <?php elseif ($action == 'returnTitleConfirm'): ?>
-        <form class="" method="POST" name="returnTitle" data-lightbox-onclose="VuFind.refreshPage">
-            <div class="pull-right">
+    <?php elseif ($action == 'editHoldConfirm'): ?>
+        <form class="" method="POST" name="editHold" data-lightbox-onclose="VuFind.refreshPage">
 
-                <input name="doAction" value="returnTitle" type="hidden">
-                <input class="btn btn-primary" name="returnTitle" value="<?=$this->transEscAttr('od_but_return'); ?>" type="submit">
+            <div class="form-group">
+                <label id="select-suspension-label"><?=$this->transEsc('od_susp_hold_confirm'); ?></label>
+                <select class="form-control" name="suspendValue" id="select-suspension" data-open="false">
+                    <option value="0" selected=""><?=$this->transEsc('od_susp_asap'); ?></option>
+                    <option value="7"><?=$this->transEsc('od_susp_after', ['%%days%%' => '7']); ?></option>
```

Comment:
Would it make sense to refactor this to:

```
<?php $durations = [7, 14, 21, 28, 60, 90, 180]; ?>
<?php foreach ($durations as $duration): ?>
  <option value="<?=$duration?>"><?=$this->transEsc('od_susp_after', ['%%days%%' => $duration]); ?></option>
<?php endforeach; ?>
```

?

That would also open the door to making this a config setting if there were any value in doing that.